hotsite/mainpage/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Convocation</title>
+
+    <link rel="stylesheet" href="style/prototype.css">
+
+    <!--Importing text font-->
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel+Decorative:wght@400;700;900&family=Cinzel:wght@400..900&family=MedievalSharp&display=swap" rel="stylesheet">
```

Comment:
You have included several fonts from Google Fonts but are only using MedievalSharp. Check if you really need the others (Cinzel, Cinzel Decorative) or if you can clean them up.